
<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01//EN">
<html>
<meta http-equiv="content-type" content="text/html; charset=UTF-8" />
<head>
   <title>Dr. Xiaoqi Zheng </title>
   <link rel="stylesheet" type="text/css" 
	href="xiaoqi.css">
   <meta name="description" content="Home page of Xiaoqi Zheng, 
   associate professor of Mathematics at Shanghai Normal University">
   <meta name="keywords" content="Machine-learning, biostatistics,
   bioinformatics, DNA methylation, cancer">
</head>



<!-- Site navigation menu -->
<ul class="navbar">
  <li><a href="index.html">Home</a>
<!--  <li><a href="CURRICULUM_VITAE.pdf">CV</a> -->
  <li><a href="research.html">Research</a>
  <li><a href="publications.html">Publications</a>
  <li><a href="people.html">Lab member</a>
  <li><a href="teaching.html">Teaching</a>
</ul>




<body>
<h2>Publications</h2>
A full list of publications can be found at my 
<a href="http://scholar.google.com/citations?user=dVaC46oAAAAJ&hl=en">Google scholar page</a>. 

<table cellspacing="0" cellpadding="5", width=100%>

<tr><td><h3>Peer reviewed journals (#: cofirst author, *: corresponding author)</h3></td></tr> 

<tr><td><u>2018</u>
<ul>
<li> D Wei, C Liu, <b>X Zheng</b>*, Y Li, Comprehensive anticancer drug response prediction based on a simple cell line-drug complex network model, <i><b> BMC Bioinformatics</b></i>, 2019, 20:44.
<li> Hu X, Zhang J, Wang J, Fu J, Li T, <b>Zheng X</b>, Wang B, Gu S, Jiang P, Fan J, Carroll, MC, Wucherpfennig K, Hacohen N, Zhang F, Zhang P, Liu JS*, Li B*, Liu XS*:  Widespread B cell clonal expansions and related mechanism of immune evasion in human cancers, i><b>Nature Genetics</b></i>, 2019, 51(3):560-567. 
<li> Xu T, <b>Zheng X</b>, Li B, Jin P, Qin Z, Wu H*: A comprehensive review of computational prediction of genome-wide features, i><b>Briefings in Bioinformatics</b></i>, doi: 10.1093/bib/bby110. 

<tr><td><u>2018</u>
<ul>
<li> H Dou, Y Fang, <b>X Zheng</b>*, Universal informative CpG sites for inferring tumor purity from DNA methylation microarray data, <i><b>Journal of Bioinformatics and Computational Biology</b></i>, 2018, 16(3):1750030.
<li> Y Qin, H Feng, M Chen, H Wu, <b>X Zheng</b>*, InfiniumPurify: An R package for estimating and accounting for tumor purity in cancer methylation research, <i><b>Genes & Diseases</b></i>, 2018, 5:43e45
<li> Fan H#, Lv P#, Huo X, Wu J, Wang Q, Cheng L, Liu Y, Tang Q, Zhang L, Zhang F, <b>Zheng X</b>, Wu H, Wen B*, The nuclear matrix protein HNRNPU maintains 3D genome architecture globally in mouse hepatocytes, <i><b>Genome Research</b></i>, 2018, 28:192–202. 

<tr><td><u>2017</u>
<ul>

<li> <b>X Zheng</b>*, N Zhang, HJ Wu, H Wu*, Estimating and accounting for tumor purity in the analysis of DNA methylation data from cancer studies, <i><b>Genome Biology</b></i>, 2017, 18:17. 
<li> <b>X Zheng</b>*, S Zhong, From structure to function, how bioinformatics help to reveal functions of our genomes, <i><b>Genome Biology</b></i>, 2017, 18:183.
<li> Zhang W#, Feng H#, Wu H*, <b>Zheng X</b>*, Accounting for Tumor purity improves cancer subtype lassification from DNA methylation data, <i><b>Bioinformatics</b></i>, 2017, 33(17), 2651–2657. 
<li> Niu B, Paulson JN, <b>Zheng X</b>, Koltera R*, Simplified and representative bacterial community of maize roots, <i><b>Proceedings of the National Academy of Sciences of the United States of America</b></i>, 2017 114 (12):E2450-E2459.


<tr><td><u>2016</u>
<ul>

<li> Ni T, Li XY, Lu N, An T, Liu ZP, Fu R, Lv WC, Zhang YW, Xu XJ, Grant Rowe R, Lin YS, Scherer A, Feinberg T, <b>Zheng XQ</b>, Chen BA, Liu XS, Guo QL1, Wu ZQ, Weiss SJ*, Snail1-Dependent p53 Repression Regulates Expansion and Activity of Tumor-Initiating Cells in Breast Cancer, <i><b>Nature Cell Biology</b></i>, 2016, 18(11):1221-1232. 

<li> X Liu, J Yang, Y Zhang, Y Fang, F Wang, J Wang, <b>X Zheng</b>* J Yang*: A systematic study on drug response associated genes using baseline gene expressions of the Cancer Cell Line Encyclopedia, <i><b>Scientific Reports</b></i>, 2016, 6:22811.

<li> Y Li, Q Liu, <b>X Zheng</b>*: DUC-Curve, a Highly Compact 2D Graphical Representation of DNA Sequences and Its Application in Sequence Alignment
, <i><b>Physica A</b></i>, 2016, 456:256-270.

<li> F Wang, N Zhang, J Wang, H Wu*, <b>X Zheng</b>*: Tumor purity and differential methylation in cancer epigenomics, <i><b>Briefings in Functional Genomics</b></i>, 2016, 15(6):408-419.

<tr><td><u>2015</u>
<ul>

<li> N Zhang#, HJ Wu#, W Zhang, J Wang, H Wu*, <b>X Zheng</b>*: Predicting tumor purity from methylation microarray data
<i><b>Bioinformatics</b></i>, 2015, 31(21):3401–3405. 
<ul>
<li><a href="TCGA.zip">Tumor purity data for TCGA samples with known 450k array by InfiniumPurify</a>
</ul>
<li> N Zhang#, H Wang#, Y Fang, J Wang*, <b>X Zheng</b>*, XS Liu*: Predicting anticancer drug responses using a duallayer
integrated cell line-drug network model, <i><b>PLoS Computational Biology</b></i>, 2015, 11:e1004498.

<li>H Wang#, <b>X Zheng</b>#, T Fei, J Wang, X Li, Y Liu, F Zhang: Towards pathway-centric cancer therapies via pharmacogenomic profiling analysis of ERK signalling pathway. 
<i><b>Clinical and translational medicine</b></i> 2015, 4(1):1-9. 	

<li> Z Dong#, N Zhang#, C Li, H Wang, Y Fang, J Wang, <b>X Zheng</b>*: Anticancer drug sensitivity prediction in cell lines from baseline gene expression through recursive feature selection
<i><b>BMC cancer</b></i>, 2015,15(1):489.	 	

<li> Y Qin, M Chen, H Wang, <b>X Zheng</b>*: A Network Flow-Based Method to Predict Anticancer Drug Sensitivity, 
<i><b>PloS one</b></i> 2015, 10(5):e0127380.

<li> Y Fang, Y Qin, N Zhang, J Wang, H Wang, <b>X Zheng</b>*: DISIS: Prediction of Drug Response through an Iterative Sure Independence Screening
<i><b>PloS one</b></i> 2015, 10(3):e0120408.


<tr><td><u>2014</u>
<ul>

<li><b>Zheng X#</b>, Zhao Q#, Wu H-J#, Li W, Wang H, Meyer CA, Qin QA, Xu H, Zang C, Jiang P*, 
Zhang Y*, Liu XS*: MethylPurify: tumor purity deconvolution and differential methylation 
detection from single tumor DNA methylomes. <i><b>Genome Biology</b></i> 2014, 15:419.
<a href="http://genomebiology.com/2014/15/8/419">[Link]</a>

<li>Li L, Yu S, Xiao W, Li Y, Huang L, <b>Zheng X</b>*, Zhou S*, Yang H*: 
Sequence-based identification of recombination spots using pseudo nucleic acid 
representation and recursive feature extraction by linear kernel SVM. 
<i><b>BMC bioinformatics</b></i> 2014, 15:340.
<a href="http://www.biomedcentral.com/1471-2105/15/340">[Link]</a>

<li>Li Y, Liu Q, <b>Zheng X</b>*, He PA: UC‐Curve: A highly compact 2D graphical 
representation of protein sequences. <i><b>International Journal of Quantum Chemistry</b></i> 2014, 
114:409-415.


<li>Li L, Yu S, Xiao W, Li Y, Hu W, Huang L, <b>Zheng X</b>*, Zhou S, Yang H: Protein submitochondria localization from integrated sequence repesentation and SVM-based backward feature extraction. <i><b>Mol BioSyst</i></b> 2014.
<li>Li L, Cui X, Yu S, Zhang Y, Luo Z, Yang H, Zhou Y, <b>Zheng X</b>*: PSSP-RFE: Accurate Prediction of Protein Structural Class by Recursive Feature Extraction from PSI-BLAST Profile, Physical-Chemical Property and Functional Annotations. <i><b>PloS one</i></b> 2014, 9:e92863.
<li>Yu X, Gao H, Zheng X, Li C, Wang J: A computational method of predicting regulatory interactions in Arabidopsis based on gene expression data and sequence information. <i><b>Computational biology and chemistry</i></b> 2014, 51:36-41.

<tr><td><u>2013</u>
<ul>

<li>Zhu J, Qin Y, Liu T, Wang J, <b>Zheng X</b>*: Prioritization of candidate disease genes by topological similarity between disease and protein diffusion profiles. <i><b>BMC bioinformatics</i></b> 2013, 14:S5.

<tr><td><u>2012</u>
<ul>

<li>Yu X, Zheng X, Meng L, Li C, Wang J: A Support Vector Machine Based Method to Predict Success for Polymerase Chain Reactions. <i><b>Combinatorial chemistry & high throughput screening</i></b> 2012, 15:486-491.
<li>Yu X, <b>Zheng X</b>*, Liu T, Dou Y, Wang J: Predicting subcellular location of apoptosis proteins with pseudo amino acid composition: approach from amino acid substitution matrix and auto covariance transformation. <i><b>Amino acids</i></b> 2012, 42:1619-1625.
<li>Liu T, Geng X, <b>Zheng X</b>*, Li R, Wang J: Accurate prediction of protein structural class using auto covariance transformation of PSI-BLAST profiles. <i><b>Amino acids</i></b> 2012, 42:2243-2249.
<li>Li Y, Qin Y, <b>Zheng X</b>*, Zhang Y: Three‐unit semicircles curve: A compact 3D graphical representation of DNA sequences based on classifications of nucleotides. <i><b>International Journal of Quantum Chemistry</i></b> 2012, 112:2330-2335.
<li>Li L, Zhang Y, Zou L, Li C, Yu B, <b>Zheng X</b>*, Zhou Y: An ensemble classifier for eukaryotic protein subcellular location prediction using gene ontology categories and amino acid hydrophobicity. <i><b>PloS one</i></b> 2012, 7:e31057.

<tr><td><u>2011</u>
<ul>


<li><b>Zheng X</b>, Liu T, Yang Z, Wang J: Large cliques in Arabidopsis gene coexpression network and motif discovery. <i><b>Journal of plant physiology</i></b> 2011, 168:611-618.
<li>Yu X, Liu T, <b>Zheng X</b>*, Yang Z, Wang J: Prediction of regulatory interactions in Arabidopsis using gene-expression data and support vector machines. <i><b>Plant Physiology and Biochemistry</i></b> 2011, 49:280-283.
<li>Wang W, Geng X, Dou Y, Liu T, <b>Zheng X</b>*: Predicting protein subcellular localization by pseudo amino acid composition with a segment-weighted and features-combined approach. <i><b>Protein and peptide letters</i></b> 2011, 18:480-487.
<li>Li C, Ma H, Zhou Y, Wang X, <b>Zheng X</b>: Similarity analysis of DNA sequences based on the weighted pseudo‐entropy. <i><b>Journal of computational chemistry</i></b> 2011, 32:675-680.
<li>Dou Y, Geng X, Gao H, Yang J, <b>Zheng X</b>, Wang J: Sequence conservation in the prediction of catalytic sites. <i><b>The protein journal</i></b> 2011, 30:229-239.


<tr><td><u>2010</u>
<ul>

<li><b>Zheng X</b>, Li C, Wang J: An information‐theoretic approach to the prediction of protein structural class. <i><b>Journal of computational chemistry</i></b> 2010, 31:1201-1206.
<li>Liu T, <b>Zheng X</b>*, Wang J: Prediction of protein structural class for low-similarity sequences using support vector machine and PSI-BLAST profile. <i><b>Biochimie</i></b> 2010, 92:1330-1334.
<li>Liu T, <b>Zheng X</b>*, Wang J: Prediction of protein structural class using a complexity-based distance measure. <i><b>Amino acids</i></b> 2010, 38:721-728.
<li>Liu T, <b>Zheng X</b>*, Wang C, Wang J: Prediction of subcellular location of apoptosis proteins using pseudo amino acid composition: an approach from auto covariance transformation. <i><b>Protein and peptide letters</i></b> 2010, 17:1263-1269.
<li>Li C, Li Z, <b>Zheng X</b>, Ma H, Yu X: A generalization of Lempel-Ziv complexity and its application to the comparison of protein sequences. <i><b>Journal of mathematical chemistry</i></b> 2010, 48:330-338.
<li>Dou Y, <b>Zheng X</b>, Yang J, Wang J: Prediction of catalytic residues based on an overlapping amino acid classification. <i><b>Amino acids</i></b> 2010, 39:1353-1361.
<li>Dou Y, <b>Zheng X</b>, Wang J: Several appropriate background distributions for entropy-based protein sequence conservation measures. <i><b>Journal of theoretical biology</i></b> 2010, 262:317-322.

<tr><td><u>2009</u>
<ul>

<li><b>Zheng X</b>, Qin Y, Wang J: A Poisson model of sequence comparison and its application to coronavirus phylogeny. <i><b>Mathematical biosciences</i></b> 2009, 217:159-166.
<li><b>Zheng X</b>, Liu T, Wang J: A complexity-based method for predicting protein subcellular location. <i><b>Amino acids</i></b> 2009, 37:427-433.
<li><b>Zheng X</b>, Li C, Wang J: A complexity-based measure and its application to phylogenetic analysis. <i><b>Journal of mathematical chemistry</i></b> 2009, 46:1149-1157.
<li><b>Zheng X</b>, Dou Y, Wang J: Phylogenetic inference from binary sequences reduced by primary DNA sequences. <i><b>Journal of mathematical chemistry</i></b> 2009, 46:1137-1148.
<li>Li C, Yu X, Yang L, <b>Zheng X</b>, Wang Z: 3-D maps and coupling numbers for protein sequences. <i><b>Physica A: Statistical Mechanics and its Applications</i></b> 2009, 388:1967-1972.
<li>Dou Y, <b>Zheng X</b>, Wang J: Prediction of catalytic residues using the variation of stereochemical properties. <i><b>The protein journal</i></b> 2009, 28:29-33.

<tr><td><u>2008</u>
<ul>


<li>Wang J, <b>Zheng X</b>*: Comparison of protein secondary structures based on backbone dihedral angles. <i><b>Journal of theoretical biology</i></b> 2008, 250:382-387.
<li>Wang J, <b>Zheng X</b>*: WSE, a new sequence distance measure based on word frequencies. <i><b>Mathematical biosciences</i></b> 2008, 215:78-83.


</table>

<p>
[ <a href="index.html">Home</a> ] 
 

</body>
</html>
